src/main/java/cc/hyperium/integrations/watchdog/ThankWatchdog.java

```diff
@@ -0,0 +1,21 @@
+package cc.hyperium.integrations.watchdog;
+
+import cc.hyperium.Hyperium;
+import cc.hyperium.config.Settings;
+import cc.hyperium.event.ChatEvent;
+import cc.hyperium.event.InvokeEvent;
+import net.minecraft.client.Minecraft;
+
+public class ThankWatchdog {
+
+  @InvokeEvent
+  public void onChat(ChatEvent e) {
+    String watchdogBanMessage = "A player has been removed from your game for hacking or abuse. Thanks for reporting it!";
```

Comment:
Why recreate this every time? Definitely just make this final and class-level.

Reply:
so just public final void?

Reply:
I meant the ban message

Reply:
oh, ok.